src/test/java/org/traccar/protocol/GatorProtocolEncoderTest.java

```diff
@@ -0,0 +1,32 @@
+package org.traccar.protocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.junit.jupiter.api.Test;
+import org.traccar.ProtocolTest;
+import org.traccar.model.Command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class GatorProtocolEncoderTest extends ProtocolTest {
+
+    @Test
+    void testEncodeId() throws Exception {
+        var encoder = inject(new GatorProtocolEncoder(null));
+        ByteBuf pseudoId = Unpooled.buffer();
+        pseudoId.writeByte(0x20);
+        pseudoId.writeByte(0x08);
+        pseudoId.writeByte(0x95);
+        pseudoId.writeByte(0x8C);
```

Comment:
Here using HEX string actually makes more sense because it's easier to see the different. In fact we usually compare strings in unit tests.

Reply:
May we remove this test as the method `encodeId` is include in `encodeContent` ? 

`encodeContent` is tested by `testEncode()`

Reply:
Sure, that's up to you.